docs/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to.md

```diff
@@ -171,6 +220,150 @@ public class ExampleClass
 The JSON value could not be converted to System.String.
 ```
 
+## Scenarios using JsonSerializer that are supported starting in 5.0
+
+### Quoted numbers
+
+::: zone pivot="dotnet-5-0"
+`Newtonsoft.Json` can serialize or deserialize numbers represented by JSON strings (surrounded by quotes). For example, it can accept: `{"DegreesCelsius":"23"}` instead of `{"DegreesCelsius":23}`. To enable that behavior in <xref:System.Text.Json>, set <xref:System.Text.Json.JsonSerializerOptions.NumberHandling%2A?displayProperty=nameWithType> to <xref:System.Text.Json.Serialization.JsonNumberHandling.WriteAsString> or <xref:System.Text.Json.Serialization.JsonNumberHandling.AllowReadingFromString>.
+
+If you're using `System.Text.Json` indirectly by using ASP.NET Core, you don't need to do anything to get behavior like `Newtonsoft.Json`. ASP.NET Core specifies [web defaults](system-text-json-how-to.md#web-defaults-for-jsonserializeroptions) when it uses `System.Text.Json`, which allows quoted numbers.
```

Comment:
Should it be "which allow" singular at the end?

Reply:
Could go either way -- "**specifies** web defaults, which allows..." or "specifies **web defaults**, which allow..."